sdk/core/azure-core/CHANGELOG.md

```diff
@@ -24,6 +25,7 @@
 
 - Throw Azure::Http::TransportException if creating new connection fails.
 - Response objects store Nullable\<T\>.
+- Calling `Cancel()` from context now throws OperationCanceledException.
```

Comment:
Just to understand, what did it throw in beta2?

```suggestion
- Calling `Cancel()` from context now throws `OperationCanceledException`.
```

Reply:
it was just doing `throw` without any specific Exception, like:
```cpp
if (CancelWhen() < std::chrono::system_clock::now())
      {
        // TODO: Runtime Exc
        throw;
      }
```
calling throw with no exception terminates the program if there is not a previous catch exception to re-throw.
See: https://stackoverflow.com/questions/5378677/what-does-a-single-throw-statement-do